src/utils/authentication.js

```diff
@@ -5,21 +5,28 @@ const signupValidation = [
   check('username')
     .isLength({ min: 2 })
     .withMessage('Your username is required'),
-  check('email').isEmail().normalizeEmail().withMessage('Please provide a valid email address'),
```

Comment:
why remove normalizeEmail here ? i think irina added it because it wasn't validating email with right format before. do you validate it somewhere else ?

Reply:
It was me who added before, normalizeEmail is only sanitisation of email and since it mutates the req it throws error in my testing therefore I removed it. express validators validates the email